index.html

```diff
@@ -272,29 +272,42 @@ <h5 class="navbar__submenu-title--right">Students</h5>
         <section>
             <div class="welcome">
                 <div class="welcome__box">
-                    <h1 class="welcome__title">GEEKS UI, Premium LMS / Education HTML Template</h1>
+                    <h1 class="welcome__title"><span class="welcome__geeks-span">Geeks UI</span>, Premium LMS / Education HTML Template</h1>
                     <p class="welcome__description">Geeks UI Premium LMS education HTML template, perfect for online courses, colleges, schools, training hubs, and corporate programs. Enhance your educational platform with seamless design and advanced features for modern learning.</p>
                 </div>
                 <div class="welcome__btn-box">
-                    <button class="welcome__button welcome__button--primary">Buy Geeks</button>
-                    <button class="welcome__button welcome__button--secondary">Preview</button>
+                    <button class="welcome__button welcome__button--primary"><i class="welcome__cart-icon fa-solid fa-cart-shopping"></i><span>Buy Geeks</span></button>
```

Comment:
Why do you have `<span>` without class here? :thinking:

Reply:
Ye, I missed this one. I'll get rid of that as `<i>` alone is enough here

Reply:
Ye, I missed this one. I'll get rid of that `span`, it is not needed here.